controllers/managedocs_controller.go

```diff
@@ -1786,3 +1795,30 @@ func (r *ManagedOCSReconciler) setDeviceSetCount(deviceSet *ocsv1.StorageDeviceS
 		deviceSet.Count = currDeviceSetCount
 	}
 }
+
+func (r *ManagedOCSReconciler) reconcileRookOverrideConfig() error {
+	if r.DeploymentType != consumerDeploymentType {
+		r.Log.Info("Non consumer deployment, skipping reconcile for rook override config")
+		return nil
+	}
+	r.Log.Info("Reconciling Rook Override Config Map")
+	_, err := ctrl.CreateOrUpdate(r.ctx, r.Client, r.rookConfigOverride, func() error {
+		if err := r.own(r.rookConfigOverride); err != nil {
```

Comment:
If we are not the ones to create it we should not own it.
Please note that if we do not own we will need to add a manual watch for this

Reply:
We are the ones creating it. Since rook will look on this at clusttr creation and apply the settings.